ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
Er, sorry right, there was this comment:

```
Root rotation should always happen by appending a new CA cert, and later removing the old one,
so it's safer to do it in this order of operations (write root, write certificate)
```

Since I think this was accounting for the possibility that the new certs are signed using the new root CA.  However, I don't think root rotation in the swarm cluster can actually happen that way, because all the nodes would have to first receive the new bundle containing both root CA certs (old and new) before the nodes can start getting new TLS keys that are signed with the new root key.

Otherwise, if not all of the nodes in the cluster have the bundle containing both root certs yet, some inter-node communication will fail.

As a standalone ca module, however, that ordering does make sense.